src/model/ScheduleDay.cpp

```diff
@@ -176,18 +181,64 @@ namespace model {
         return 0.0;
       }
 
+      TimeSeries ts = this->timeSeries();
+
+      DateTimeVector dateTimes = ts.dateTimes();
+      Vector values = ts.values();
+
+      for (unsigned i = 0; i < dateTimes.size() - 1; ++i) {
+        openstudio::Time t0 = dateTimes[i].time();
+        openstudio::Time t1 = dateTimes[i + 1].time();
+        if (t1.totalDays() == 0.0) {  // this is 00:00:00 from the next day
+          t1 = openstudio::Time(0, 24, 0);
+        }
+
+        if (t0 == time) {
+          double value = values[i];
+        } else if (t1 == time) {
+          double value = values[i + 1];
+        } else if ((t0 < time) && (time < t1)) {
+          double value = values[i + 1];
+        }
```

Comment:
Should we still linearly interpolate for requested times _between_ timesteps? Seems like E+ doesn't see/use them, so we return the value corresponding to the following timestep?

Reply:
Could we just use `HoldNextInterp` here?